test/java/org/opendatakit/briefcase/ui/reused/NopAnalytics.java

```diff
@@ -0,0 +1,69 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import java.awt.event.ComponentEvent;
+import java.awt.event.ComponentListener;
+import javax.swing.JComponent;
+
+public class NopAnalytics extends Analytics {
```

Comment:
I wonder if this wouldn’t sound more clear as NoOpAnalytics. I’ve never heard No-Op or NOP pronounced as a single syllable before. (Maybe you’re thinking spell it NopAnalytics and pronounce it no-op-analytics.)

Reply:
I've seen both... Honestly, I don't have a strong opinion. Let's be explicit and change it to `NoOp` everywhere...